examples/css3d_molecules.html

```diff
@@ -206,9 +206,21 @@
 				var b_b = document.getElementById( "b_b" );
 				var b_ab = document.getElementById( "b_ab" );
 
-				b_a.addEventListener( 'click', function() { visualizationType = 0; showAtoms() } );
-				b_b.addEventListener( 'click', function() { visualizationType = 1; showBonds() } );
-				b_ab.addEventListener( 'click', function() { visualizationType = 2; showAtomsBonds() } );
+				b_a.addEventListener( 'click', function () {
+
+					visualizationType = 0; showAtoms();
```

Comment:
2 statements on 1 line??

Reply:
Hmm, looks like the auto eslint error fixer is missing these.